daffodil-test/src/test/scala-debug/edu/illinois/ncsa/daffodil/section00/general/TestUnparserGeneralDebug.scala

```diff
@@ -57,5 +57,5 @@ class TestUnparserGeneralDebug {
   @Test def test_puaInfosetChars_03() { runner.runOneTest("puaInfosetChars_03") }
   @Test def test_puaInfosetChars_04() { runner.runOneTest("puaInfosetChars_04") }
 
-  @Test def test_bitOrderOVC1() = { runner2.runOneTest("bitOrderOVC1") }
+  @Test def test_bitOrderOVC1() = { runner2.trace.runOneTest("bitOrderOVC1") }
```

Comment:
This test doesn't yet pass, btw. Probably should fix this (which is DAFFODIL-1843), while in this area of the code. If this test doesn't work, it's surprising that Link16 works.

Reply:
passes now.